internal/distro/rhel9/images.go

```diff
@@ -301,7 +301,12 @@ func edgeInstallerImage(workload workload.Workload,
 	img.Groups = users.GroupsFromBP(customizations.GetGroups())
 
 	img.SquashfsCompression = "xz"
-	img.AdditionalDracutModules = []string{"prefixdevname", "prefixdevname-tools"}
+	img.AdditionalDracutModules = []string{
+		"prefixdevname",
+		"prefixdevname-tools",
+		"nvdimm", // non-volatile DIMM firmware (provides nfit, cuse, and nd_e820)
```

Comment:
This array isn't sorted. I wonder if we want to sort this out in this PR. :thinking:

Reply:
yeah good idea.  I think I might sort the base dracut modules as well.  Coming in a separate commit.

Reply:
Not sorting the other dracut modules.  It will change all the installer manifests in all distros and I don't want to add noise to this PR.